src/main/scala/zio/redis/options/zadd.scala

```diff
@@ -0,0 +1,16 @@
+package zio.redis.options
+
+object zadd {
+  sealed trait Updates
+
+  object Updates {
+    case object XX extends Updates
+    case object NX extends Updates
+  }
+
+  case object CH
+  case object INCR
+
+  type CH   = CH.type
```

Comment:
I like and recommend this pattern. 👍

Reply:
There's a slight improvement on it:

```scala
sealed trait CH
val CH: CH = new CH
```

This way the value `CH` has type `CH`.